src/matcha_ml/runners/azure_runner.py

```diff
@@ -42,6 +45,38 @@ def _show_terraform_outputs(self, matcha_state: MatchaState) -> None:
         resources_json = dict_to_json(resources_dict)
         print_json(resources_json)
 
+    def is_local_state_stale(self) -> bool:
+        """Checks for congruence between the local config file and the local tfvars file."""
+        local_tfvars_file = os.path.join(
+            os.getcwd(),
+            ".matcha",
+            "infrastructure",
+            "remote_state_storage",
+            "terraform.tfvars.json",
+        )
+        local_config_file = os.path.join(os.getcwd(), "matcha.config.json")
+        if os.path.exists(local_tfvars_file) and os.path.exists(local_config_file):
+            with open(local_tfvars_file) as tf:
```

Comment:
Nit: Watch spacing - you could space this code out of a bit to make it much more readable